exchanges/ftx/ftx_wrapper.go

```diff
@@ -1061,7 +1056,10 @@ func (f *FTX) GetHistoricCandlesExtended(p currency.Pair, a asset.Item, start, e
 		Interval: interval,
 	}
 
-	dates := kline.CalculateCandleDateRanges(start, end, interval, f.Features.Enabled.Kline.ResultLimit)
+	dates, err := kline.CalculateCandleDateRanges(start, end, interval, f.Features.Enabled.Kline.ResultLimit)
+	if err != nil {
+		return kline.Item{}, err
+	}
```

Comment:
In order to test the missing data functionality. I used the TestGetHistoricCandlesExtended test (without modifying it). However a few lines below where I have commented this in f.GetHistoricalData function call, I did time.Add(-24*time.Hour) on the endtime. The output I got was perfect which showed that i was missing data on the last day however there is some funky stuff going on with 2019-11-14 - 2019-11-15 
![image](https://user-images.githubusercontent.com/31364354/121296786-8dd59800-c934-11eb-9cfd-1dd7dcc9a664.png)

Reply:
Just gave the backstory here incase you wanted to reproduce this scenario. Not sure if this is your fault yet but just wondering as to why thats happening

Reply:
My fault actually. This is in a forloop and it sends 2 requests so we should technically expect 2 days of data missing. Resolving it now